app/domain/authentication/authn_azure/gather_request_parameters.rb

```diff
@@ -0,0 +1,48 @@
+require 'command_class'
+
+module Authentication
+  module AuthnAzure
+
+    GatherRequestParameters = CommandClass.new(
```

Comment:
i don't think we need a command class for this. the logic here can be performed in a method in the calling class.